packages/site-client/src/components/wco-catalog-page.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ import {html, css, LitElement} from 'lit';
+ import {customElement} from 'lit/decorators.js';
+ import './wco-top-bar.js';
+ import './wco-catalog-search.js';
+ 
+ @customElement('wco-catalog-page')
+ export class WCOCatalogPage extends LitElement {
+   static styles = css`
+     :host {
+       display: block;
+     } 
+   `;
+ 
+   render() {
+     return html`
+       <wco-top-bar></wco-top-bar>
+       <h1>Catalog</h1>
+       <wco-catalog-search></wco-catalog-search>
+     `;
+   }
+ }
+ 
+ declare global {
+   interface HTMLElementTagNameMap {
+     'wco-catalog-page': WCOCatalogPage;
+   }
+ }
+ 
```

Comment:
Very nit, but files should generally end in newlines, I think? I always enable "Insert Final Newline" in vscode. Maybe just an outdated habit, but not including it can make certain shell operations a bit weird, e.g. if you `cat` two files together, the last line of the first file will be prepended to the first line of the second file.

Reply:
Oops, I hadn't formatted and the whole file has an extra leading space for some reason.